### PR TITLE
ALAMODE + dep

### PR DESCRIPTION
**Pull request by c-gr34sl3y**
For INC1401671 - `ALAMODE-1.4.2-foss-2022b.eb`

Dep:
* `spglib-2.0.2-GCCcore-11.3.0.eb` modified from upstream easyconfig

* [ ] Assigned to reviewer

Default:
* [ ] EL8-icelake
* [ ] EL8-cascadelake
* [ ] EL8-haswell

### CODE REVIEW

File: easyconfigs/a/ALAMODE/ALAMODE-1.4.2-foss-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    "-DUSE_MKL_FFT=no",
+])
+
+local_binary_files = [
+    "alm/alm", "anphon/anphon", "tools/analyze_phonons", "tools/dfc2", "tools/qe2alm",
+    "tools/analyze_phonons.py", "tools/extract.py", "tools/plotdos.py", "tools/displace.py",
+    "tools/plotband.py",
+]
+
+files_to_copy = [([x for x in local_binary_files], 'bin'), (['docs', 'example'], '')]
```
Comment (branfosj) [inline easyconfigs/a/ALAMODE/ALAMODE-1.4.2-foss-2022b.eb:49]:
```suggestion
files_to_copy = [([x for x in local_binary_files], 'bin'), 'docs', 'example']
```

File: easyconfigs/a/ALAMODE/ALAMODE-1.4.2-foss-2022b.eb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    ('Boost', '1.81.0'),
+    ('Eigen', '3.4.0'),
+    ('spglib', '2.0.2'),
+    ('FFTW', '3.3.10'),
+    ('HDF5', '1.14.0'),
+    ('Python', '3.10.8'),
+]
+
+configopts = ' '.join([
+    "-DCMAKE_BUILD_TYPE=Release",
```
Comment (branfosj) [inline easyconfigs/a/ALAMODE/ALAMODE-1.4.2-foss-2022b.eb:38]: Is the default, so should be removed.

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easyconfigs/a/ALAMODE/ALAMODE-1.4.2-foss-2022b.eb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    "-D_HDF5=yes",
```
Comment (branfosj) [inline easyconfigs/a/ALAMODE/ALAMODE-1.4.2-foss-2022b.eb:39]: Is this correct? It looks odd.

File: easyconfigs/s/spglib/spglib-2.0.2-GCCcore-11.3.0.eb [hunk 4]
```diff
@@ -0,0 +1,31 @@
+easyblock = 'CMakeMake'
+
+name = 'spglib'
+version = '2.0.2'
+
+homepage = 'https://spglib.github.io/spglib/'
+description = """Spglib is a C library for finding and handling crystal symmetries."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
```
[the remark this answers is not in the document]
Comment (c-gr34sl3y) [reply easyconfigs/s/spglib/spglib-2.0.2-GCCcore-11.3.0.eb]: Thanks - fixed in f5082de

File: easyconfigs/a/ALAMODE/ALAMODE-1.4.2-foss-2022b.eb (hunk 2 above)
Comment (c-gr34sl3y) [reply easyconfigs/a/ALAMODE/ALAMODE-1.4.2-foss-2022b.eb]: done - fixed in a031b01

File: easyconfigs/a/ALAMODE/ALAMODE-1.4.2-foss-2022b.eb (hunk 3 above)
Comment (c-gr34sl3y) [reply easyconfigs/a/ALAMODE/ALAMODE-1.4.2-foss-2022b.eb]: Yes it should - I erroneously used the make option here.
fixed in a031b01

File: easyconfigs/a/ALAMODE/ALAMODE-1.4.2-foss-2022b.eb (hunk 1 above)
Comment (c-gr34sl3y) [reply easyconfigs/a/ALAMODE/ALAMODE-1.4.2-foss-2022b.eb]: done - updated in a031b01
the tuple wasn't required

**Review by branfosj — APPROVED** [review]

**Pull request merged**